test/helpers/utils.go

```diff
@@ -247,6 +247,24 @@ func (kubectl *Kubectl) WaitCiliumEndpointReady(podFilter string, k8sNode string
 	return ciliumPod, endpoints
 }
 
+// WaitKafkaBroker waits for the broker to be ready, by executing
+// a produce request on existing topics and waiting for a response from broker
+func WaitKafkaBroker(pod string, cmd string, node *SSHMeta) error {
+	fmt.Print("in WaitKafkaBroker")
```

Comment:
Remove.

Reply:
You can't use fmt.Print in ginkgo. Logger please.

Reply:
done